models/tokens/bnb/tokens_bnb_bep20.sql

```diff
@@ -0,0 +1,5523 @@
+{{ config( alias='bep20')}}
+
+SELECT LOWER(contract_address) AS contract_address, symbol, decimals
+  FROM (VALUES 
+  
+('0x111111111117dc0aa78b770fa6a738034120c302', '1INCH', 18)
+,('0xfb6115445bff7b52feb98650c87f44907e58f802', 'AAVE', 18)
```

Comment:
general comment: thank you for the left comma 🫡 
it also helps avoid the extra comma at the end when copy/pasting additional tokens and forgetting to remove it

Reply:
i don't know you but i already like you